docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,80 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* Improve client protocol throughput by introducing the spooling protocol.
+  ({issue}`24214`)
+* Improve performance of querying `system.jdbc.tables` for multiple catalogs.
+  ({issue}`24159`)
+* Add support for connectors to improve performance for `MERGE` statements by
+  only changing updated columns. ({issue}`24075`)
+* Add support for changing field types inside a map. ({issue}`24248`)
+* Replace Python-based launcher script with fully-compatible, go-based
+  executable. Removes the requirement for a Python installation on Trino cluster
+  nodes. ({issue}`24271`)
+* Improve error messages for `INTERVAL` literals. ({issue}`23002`)
+
+## Security
+
+* Add support [data access control with Apache
+  Ranger](/security/apache-ranger-access-control), including support for
+  column-masking, row-filtering, and audit logging. ({issue}`22675`)
+
+## JDBC driver
+
+* Automatically use the spooling protocol, when configured on the Trino cluster,
+  and add the parameter `encoding` to optionally set the preferred encoding from
+  the JDBC driver. ({issue}`24214`)
+* Improve performance of decompression when using Java 22 or newer as runtime
+  for the JDBC driver. ({issue}`24263`)
+
+## Docker image
+
+* Replace Python-based launcher script with fully-compatible, go-based
```

Comment:
Is there a user impact for this? Or does the Docker image work just the same with this change being under the hood?

Reply:
Well.. it is smaller and has less CVEs and some people get into the container and try to run Python stuff ..

Reply:
Technically it should be transparent but nevertheless I think we have to mention it.

Reply:
Then we should just say "Remove Python from the Docker image".